rpc/src/v1/helpers/dispatch.rs

```diff
@@ -396,8 +410,13 @@ impl Dispatcher for LightDispatcher {
 		}))
 	}
 
-	fn sign(&self, accounts: Arc<AccountProvider>, filled: FilledTransactionRequest, password: SignWith)
-		-> BoxFuture<WithToken<SignedTransaction>>
+	fn sign(
+		&self,
+		accounts: Arc<AccountProvider>,
+		filled: FilledTransactionRequest,
+		password: SignWith,
+		_: impl PostSign + 'static
```

Comment:
Why is post sign ignored for light clients? That's completely breaking sending transactions.